tests/components/group/test_init.py

```diff
@@ -62,13 +67,13 @@ def test_setup_group_with_non_groupable_states(self):
             self.hass, "chromecasts", ["cast.living_room", "cast.bedroom"]
         )
 
-        assert STATE_UNKNOWN == grp.state
+        assert grp.state is None
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6259301)